src/app/components/LinkedData/index.jsx

```diff
@@ -1,117 +1,117 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import Helmet from 'react-helmet';
-import { arrayOf, string, shape } from 'prop-types';
-import propTypeCheck from '../Metadata/propTypesCheck';
+import { string, shape, arrayOf, bool } from 'prop-types';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { RequestContext } from '#contexts/RequestContext';
+import getAboutTagsContent from './getAboutTagsContent';
 
 const LinkedData = ({
-  brandName,
+  showAuthor,
   type,
-  seoHeadline,
-  firstPublished,
-  lastUpdated,
-  logoUrl,
-  publishingPrinciples,
-  noBylinesPolicy,
-  about,
-  canonicalLink,
+  seoTitle,
+  headline,
+  datePublished,
+  dateModified,
+  aboutTags,
 }) => {
-  const imgObject = 'ImageObject';
-  const newsMediaOrg = 'NewsMediaOrganization';
-  const webPageType = 'WebPage';
+  const {
+    brandName,
+    publishingPrinciples,
+    defaultImage,
+    noBylinesPolicy,
+  } = useContext(ServiceContext);
+  const { canonicalNonUkLink } = useContext(RequestContext);
```

Comment:
Since this is using contexts, it should become a container. So moving this file from `src/app/components/LinkedData/index.jsx` to `src/app/containers/LinkedData/index.jsx` would be good.